### ISSUE TITLE
Popout discord chats

### ISSUE THREAD

**Issue by Cocaine-Potato** [Vencord/plugin-requests#429]
### Plugin Description

a plugin that lets you pop out specific discord chats, so you can write in the chats like dms or smth like that while in another window, so discord doesnt use up 20% of your screen all the time (the chat popup should be simple like in this image: ![image](https://github.com/Vencord/plugin-requests/assets/143091775/d93d8d1c-c775-4009-a26b-0d96e3ce25e2) so theres no senseless stuff bloating the popout to make it bigger, and it should be resizable with an option to pin it "on top" so it doesnt disappear while on other windows (smth like the windows timer app has)

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by DarkCorpseX** [comment]
So like there's no other stuff like pinned messages and all that or just a separate DM window?

**Comment by Cocaine-Potato** [comment]
myb that could be changed in the settings, im fine with whatever whoever wants to make this into a plugin prefers

**Comment by Sqaaakoi** [comment]
this is literally a 2 year old experiment afaik

**Comment by Sqaaakoi** [comment]
horror i think they removed it (i didnt find it by searching)

**Comment by Panniku** [comment]
![image](https://github.com/user-attachments/assets/47209fbe-d395-406a-963a-21deb29538db)
still exists

**Comment by Cocaine-Potato** [comment]
id say make that an option in the plugin settings, but rn my idea was for
ut to be just a text bar on the bottom and the chat ontop (not even the
header where it says the persons name cuz that isnt needed cuz its alrdy in
the messages)

**Comment by Panniku** [comment]
fair enough, this works but doesnt inject vencord and has broken css props

**Comment by Cocaine-Potato** [comment]
in the end it is just a discord experiment xD
still hoping for some dev to see this so someone potent can make this
actually real unlike dc
